### ISSUE TITLE
Change API URL to dataset-viewer.huggingface.co?

### ISSUE THREAD

**Issue by severo** [huggingface/dataset-viewer#2666]
Follow-up to https://github.com/huggingface/dataset-viewer/issues/2650

Should we do it?
- https://github.com/huggingface/dataset-viewer/issues/2650#issuecomment-2040217875
- https://github.com/huggingface/moon-landing/pull/9520#issuecomment-2040220911

If we change it, we would have to update:
- moon-landing
- datasets
- the docs (hub, datasets, dataset-viewer)
- other written support (blog, observable, notion...)

**Comment by julien-c** [comment]
voting no for now

**Comment by severo** [comment]
yes, it's a lot of complication on infra's side, for little improvement. Closing for now.
